CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [8.2.1] 2020-03-20
+## [8.2.1] - 2020-03-20
```

Comment:
Don't know if that doesn't break some links in releases. @MarcelMue @tfussell ?

Reply:
It shouldn't - this hasn't been linked anywhere yet AFAIK